docs/modules/ROOT/pages/references/secrets.adoc

```diff
@@ -206,3 +206,28 @@ stringData:
 
 NOTE: Not all fields apply to all providers
 // end::kafka[]
+
+== Keycloak
+
+The example secrets on this page contains the plaintext values for the given keys.
+
+// tag::keycloak[]
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keycloak-creds
+stringData:
+  KEYCLOAK_HOST: "example.keycloak.com" # <1>
+  KEYCLOAK_PASSWORD: "my-password" # <2>
+  KEYCLOAK_USERNAME: "admin" # <3>
+
+----
+<1> Host to connect to
```

Comment:
Maybe specify, that this contains the k8s internal hostname. Useful when connecting from within the cluster.

The external domain is user provided.